app/facades/trip_legs_facade.rb

```diff
@@ -0,0 +1,35 @@
+class TripLegsFacade
+  def initialize(trip)
+    @trip = trip
+  end
+
+  def response
+    {
+      data:
+        {
+          places: places,
+          legs: legs
+        }
+    }
+  end
+
+  def places
+    trip.pois.map do |poi|
+      # This is only current weather,
+      # will need to update with timing between pois
+      place = PoiSerializer.new(poi).to_json_with_timing(0)
+
+      sequence_number = TripPoi.find_by(trip: trip, poi: poi).sequence_number
```

Comment:
This is good for now, I wonder if we add some `sequence_number` logic to the `PoiSerializer`

Or maybe the `trip_poi` has a `to_json` method which adds the sequence number to the `PoiSerializer` output?

Reply:
Agreed. I think the more work we push to the PoiSerializer will reduce repeition